src/linker/Linker.Steps/MarkStep.cs

```diff
@@ -2372,16 +2372,35 @@ protected virtual void MarkReflectionLikeDependencies (MethodBody body)
 				if (methodCalled == null)
 					continue;
 
-				var methodCalledDefinition = methodCalled.Resolve ();
-				if (methodCalledDefinition == null)
-					continue;
+				MarkReflectionLikeDependencyForMethodCall (body, methodCalling, instructions, i, methodCalled);
+			}
+		}
 
-				var methodCalledType = methodCalled.DeclaringType;
+		private void MarkReflectionLikeDependencyForMethodCall (
```

Comment:
Existing code prefers not to have redundant modifiers. Also, keep the declaration at one line

Reply:
Removed the modifiers and I think I fixed all the multi-line declarations